Functions/xASL_adm_PaddedString.m

```diff
@@ -0,0 +1,77 @@
+function [resultText] = xASL_adm_PaddedString(textToPrint,SymbolToFill,Color,newLines,printImmediately)
```

Comment:
I like `xASL_adm_PadedString` a lot, though I wonder if we should change its name into something that clarifies that this function is used throughout ExploreASL's modules for denoting new processing sections

Reply:
Yes, if it has this unique purpose, we can call it differently...